origin-dapp/src/utils/user.js

```diff
@@ -13,3 +15,23 @@ const { web3 } = origin.contractService
 export function formattedAddress(string) {
   return web3.utils.toChecksumAddress(string)
 }
+
+
+export function truncateAddress(address = '') {
+  if (address.length <= MAX_ADDRESS_LENGTH) return address
+  const separator = '...'
+  const frontChars = 5
+  const backChars = 4
+
+  return address.substr(0, frontChars)
+    + separator
+    + address.substr(address.length - backChars)
+}
+
+export function abbreviateName(party) {
+  const { profile = {}, fullName } = party
+  const { firstName = '', lastName = '' } = profile
+  const lastNameLetter = lastName.length && `${lastName.charAt(0)}.`
+
+  return fullName && `${firstName} ${lastNameLetter}`
```

Comment:
@Natasha08 I think that we will get `false` or `0` in the name if one of those values is `undefined`.

Reply:
@micahalcorn I have tested with an account with no names, but I will double check